### PR TITLE
DOC: Remove 1.2 versionadded and changed

### PR DESCRIPTION
**Pull request by phofl**
Lets see if the docs build like this

### CODE REVIEW

**Review by attack68 — APPROVED** [review]

File: pandas/io/parsers/readers.py [hunk 1]
```diff
@@ -365,17 +358,6 @@
...
-    .. versionchanged:: 1.2
-
-       When ``encoding`` is ``None``, ``errors='replace'`` is passed to
-       ``open()``. Otherwise, ``errors='strict'`` is passed to ``open()``.
-       This behavior was previously only the case for ``engine='python'``.
-
-    .. versionchanged:: 1.3.0
-
-       ``encoding_errors`` is a new argument. ``encoding`` has no longer an
-       influence on how encoding errors are handled.
```
Comment (rhshadrach) [inline pandas/io/parsers/readers.py:377]: Accidential?

Comment (phofl) [reply pandas/io/parsers/readers.py:377]: No, 1.3 is next and this note doesn't make much sense without the note before

**Comment by rhshadrach** [comment]
Was curious about recent PyPI downloads - here they are for the last 30 days and 365 days respectively.

```
# Last 30 days
   version  num_downloads       pct
0    2.1.3       21561118  0.141034
1    1.3.5       19570609  0.128014
2    2.0.3       17444849  0.114109
3    1.5.3       14027672  0.091757
4    2.1.4        9856006  0.064469
5    1.1.5        9464289  0.061907
6    1.2.5        6267305  0.040995
7    1.0.5        5987313  0.039164
8    1.3.4        4047997  0.026478
9    1.5.1        4011468  0.026240
10   1.5.2        2842917  0.018596
11  0.24.2        2458297  0.016080
12   2.1.0        2269155  0.014843
13   2.0.2        2101255  0.013745
14   2.1.1        1922434  0.012575
15   1.2.0        1855862  0.012139
16   1.4.2        1700640  0.011124
17   1.3.0        1621694  0.010608
18   1.4.4        1591119  0.010408
19   1.2.4        1556250  0.010180

# Last 365 days
   version  num_downloads       pct
0    1.3.5      233335844  0.153864
1    1.5.3      167123208  0.110203
2    1.1.5      139692671  0.092115
3    2.0.3      124116990  0.081844
4    1.2.5       70976974  0.046803
5    1.0.5       54345302  0.035836
6    1.3.4       45642029  0.030097
7    1.5.2       45600828  0.030070
8    1.5.1       39135032  0.025806
9   0.24.2       37744849  0.024889
10   2.0.2       35169880  0.023191
11   2.1.1       34929169  0.023033
12   2.0.1       32167209  0.021211
13   2.1.3       28439822  0.018754
14   1.0.1       24729368  0.016307
15   1.4.4       23120883  0.015246
16  0.25.3       22664845  0.014945
17   2.0.0       21441925  0.014139
18   1.4.2       21244355  0.014009
19   2.1.0       20912152  0.013790
```

**Comment by phofl** [comment]
We can't keep this notes around forever, folks can just use the version switch if they are still back on 1.3. 1.3 was released 2 years ago and this will land in the 5th release since then

File: pandas/io/parsers/readers.py (hunk 1 above)
Comment (rhshadrach) [reply pandas/io/parsers/readers.py:377]: I see - encoding_errors still has a versionadded below. Thanks.

**Review by rhshadrach — APPROVED** [review]
lgtm

**Comment by rhshadrach** [comment]
Thanks @phofl

**Pull request merged**